Premier league.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<ol><li>There could be also a draw in the game of cycle <code>x+1</code>  ! Fix that please</li><li>The condition in line <code>20</code>  is not enough - you need to verify that one of them won and the other lost.</li><li>Rename the variable <code>if_home_last_win</code>  </li><li>Use the <code>zip</code>  trick</li><li>Do not use <code>^</code>  for boolean values. Simply use <code>or</code>  </li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<p>5.	But I need that only one of them won in last cycle, not both of them. and "xor" do that most good, if I will use "or" the condition will be very long and not easy to read.</p>

Reply:
<p>I see, plus one for creativity.</p><p>Avoid ^ for logic - it is not intuitive.</p><p> </p><p>So my comment 1 is wrong, but comment 2 is still critical.</p><p> </p>